### ISSUE TITLE
Helm module `valueFiles` not loading custom values into build context.

### ISSUE THREAD

**Issue by davidjeddy** [garden-io/garden#1638]
## Bug

When using the Helm module we have values.yaml overrides that need to be applied during `helm template` and `helm apply`. However, it would appear the custom values file is not being copied into the build context.

### Current Behavior

When Garden executed `helm ... template ...` the custom values.yaml has yet to be copied into the build context. This causes a failed build.

![image](https://user-images.githubusercontent.com/6232455/75187045-3c8d9f00-5717-11ea-85b6-805786dab6cd.png)

### Expected behavior

The custom values YAML should be available in the build contexts before helm is executed.

### Workaround

We tried manually copying the custom values yaml into the build context to no avail.

### Suggested solution(s)

Copy module values before executing `helm ...`.

### Additional context

n/a

### Your environment

Ubuntu 18.04 local host
GCP GKE remote cluster

### Versions

```sh
./garden on  devel_1 [!?] 
❯ garden version
0.11.4

./garden on  devel_1 [!?] took 5s 
❯ kubectl version
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.0", GitCommit:"ddf47ac13c1a9483ea035a79cd7c10005ff21a6d", GitTreeState:"clean", BuildDate:"2018-12-03T21:04:45Z", GoVersion:"go1.11.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.10-gke.17", GitCommit:"bdceba0734835c6cb1acbd1c447caf17d8613b44", GitTreeState:"clean", BuildDate:"2020-01-17T23:10:13Z", GoVersion:"go1.12.12b4", Compiler:"gc", Platform:"linux/amd64"}

./garden on  devel_1 [!?] 
❯ docker version
Client: Docker Engine - Community
 Version:           19.03.6
 API version:       1.40
 Go version:        go1.12.16
 Git commit:        369ce74a3c
 Built:             Thu Feb 13 01:27:49 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.6
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.16
  Git commit:       369ce74a3c
  Built:            Thu Feb 13 01:26:21 2020
  OS/Arch:          linux/amd64
  Experimental:     true
 containerd:
  Version:          1.2.10
  GitCommit:        b34a5c8af56e510852c35414db4c1f4fa6172339
 runc:
  Version:          1.0.0-rc8+dev
  GitCommit:        3e425f80a8c931f88e6d94a8c831b9d5aa481657
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

**Comment by edvald** [comment]
Thanks for the report @davidjeddy! 

As it happens we just today merged a related fix: #1637 
That would have caught this, but we should also more explicitly include the value files in the default module includes.

You can in the meantime work around it by explicitly setting the `include` field to include the value files, and any other files needed (if applicable):

```
...
include: ["*.yaml"]
...
```

**Comment by davidjeddy** [comment]
Hello @edvald thats awesome! Thank you fro the prompt rely and point to the fix.
